gladier_xpcs/flows/flow_eigen.py

```diff
@@ -30,11 +37,11 @@ class XPCSEigen(ContainerBaseClient):
     }
 
     gladier_tools = [
-        'gladier_xpcs.tools.TransferFromClutchToTheta',
-        'gladier_xpcs.tools.PrePublish',
-        'gladier_xpcs.tools.AcquireNodes',
-        'gladier_xpcs.tools.EigenCorr',
-        'gladier_xpcs.tools.MakeCorrPlots',
-        'gladier_xpcs.tools.gather_xpcs_metadata.GatherXPCSMetadata',
-        'gladier_xpcs.tools.Publish',
+        TransferFromClutchToTheta,
+        PrePublish,
+        AcquireNodes,
+        EigenCorr,
+        MakeCorrPlots,
+        GatherXPCSMetadata,
+        Publish,
```

Comment:
Looks like the changes from #110 made it in here, you can remove them with a `git rebase -i head^^^`